sunpy_sphinx_theme/conf.py

```diff
@@ -21,24 +22,18 @@ def page_url(page):
     return urljoin(sunpy_website_url_base, page)
 
 
-def get_html_theme_path():
-    """
-    Return list of HTML theme paths.
-    """
-    theme_path = Path(__file__).parent.resolve()
-    return [str(theme_path)]
+html_theme = "sunpy"
+theme_base_path = get_html_theme_path()
 
+img_dir = theme_base_path / "static" / "img"
+html_favicon = str(img_dir / "favicon-32.ico")
+html_logo = png_icon = str(img_dir / "sunpy_icon_128x128.png")
+svg_icon = str(img_dir / "sunpy_icon.svg")
 
-html_theme = "sunpy"
-html_theme_path = get_html_theme_path()
-templates_path = [os.path.join(html_theme_path[0], html_theme, "templates")]
-html_static_path = [os.path.join(html_theme_path[0], html_theme, "static")]
-html_css_files = [os.path.join(html_static_path[0], "sunpy_style.css")]
-html_extra_path = [os.path.join(html_static_path[0], "img")]
-html_favicon = os.path.join(html_extra_path[0], "favicon-32.ico")
+# TODO: Remove these after downstream repos have been fixed to not rely on them
+html_static_path = [str(theme_base_path / "static")]
+html_extra_path = [str(theme_base_path / "static" / "img")]
```

Comment:
Don't know if you want to keep this hack this or patch the downstream projects. Seems easier to do this even though I don't technically think it's needed and relying on it downstream is probably a little fragile.